README.md

```diff
@@ -41,9 +42,21 @@ You will only need to run this command when dependencies change in `package.json
 
 We use npm scripts and [Webpack][] as our build system.
 
+### Microservices Overview
+
+Podium uses microservices. There are two services in this repository, the uaa service and the gateway service. There is a third the [Podium Registry][] that has its own repository and needs to be running for the others to work.
+The services have to be started in the following order:
+
+1. Podium-Registry
+2. Podium-UAA
+3. Podium-Gateway
 
 ### Running Podium for development
 
+First everything should be installed by running the following in the root folder.
+
+    mvn clean install
+
 Before the uaa and gateway services can start [Podium Registry][] needs to be up and running.
```

Comment:
Pls provide link to podium registry repo

Reply:
The links are provided at the bottom and filled in automatically